### DISCUSSION TITLE
Maybe scale down images

### DISCUSSION THREAD

**Discussion by ParetoOptimalDev** [magit/forge#600] [Ideas]
-  I viewed https://github.com/NixOS/nixpkgs/issues/258836 within forge
 - I hit C-v to scroll down, wait a second, it scrolls back up
 - reproduce by repeating the above steps using forge

Can we limit the size of images to a certain width similar to `org-actual-image-width`:

```
org-image-actual-width is a variable defined in ‘org.el’.

Its value is t

When non-nil, use the actual width of images when inlining them.

When set to a number, use imagemagick (when available) to set the
image’s width to this value.

When set to a number in a list, try to get the width from any
#+ATTR.* keyword if it matches a width specification like

  #+ATTR_HTML: :width 300px

and fall back on that number if none is found.

When set to nil, try to get the width from an #+ATTR.* keyword
and fall back on the original width if none is found.

When set to any other non-nil value, always use the image width.

This requires Emacs >= 24.1, built with imagemagick support.

  This variable was introduced, or its default value was changed, in
  version 24.4 of Emacs.
  You can customize this variable.

[back]
```

**Comment by tarsius** [comment]
I assume that you can move past the image using a moving command instead of a scrolling command. Tell me if that is not the case.

Scaling down images might make sense, but implementing that isn't a priority.

**Comment by ParetoOptimalDev** [comment]
You are correct. You can move past using the moving command.

**Comment by tarsius** [comment]
To be able to focus on other projects, I have to reduce the time I spend on Forge. That involves deciding to not implement certain features and I'm afraid this is one of those. Sorry.
